src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -13,6 +13,7 @@ import {ANONYMOUS, ANON_PM, ANON_POST_NOTIFICATION} from '../../../hooks/core/co
 import useAnonymousChannelListScreenHook from '../../../hooks/screen/useAnonymousChannelListHook';
 import useRootChannelListHook from '../../../hooks/screen/useRootChannelListHook';
 import Search from '../../ChannelListScreen/elements/Search';
+import CommunityChannelItem from '../../../components/ChatList/CommunityChannelItem';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `CommunityChannelItem` is a new addition and seems to be used correctly in the context of this file. It's replacing the previous `MessageChannelItem` for 'ANON_TOPIC' channel type.
```diff
+ import CommunityChannelItem from '../../../components/ChatList/CommunityChannelItem';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->